ddtrace/contrib/redis/asyncio_patch.py

```diff
@@ -44,7 +44,7 @@ async def _run_redis_command_async(span, func, args, kwargs):
         if redis_command in ROW_RETURNING_COMMANDS:
             span.set_metric(db.ROWCOUNT, 0)
         raise
-    finally:
+    else:
         if redis_command in ROW_RETURNING_COMMANDS:
```

Comment:
nit: `parsed_command`, `redis_command`, and `result` are all initialized in the `try-except`. Can we move this up?

Reply:
I move the code. And switch the target branch to 2.x.